app/models/wishlist.rb

```diff
@@ -0,0 +1,4 @@
+class Wishlist < ApplicationRecord
+  validates :name, length: { in: 3..30 }
```

Comment:
No problem with having a longer then 30 chars name. No point in restricting it. Just require it to be longer then 3.